### PR TITLE
[feat]tools-v2: add bs update volume flatten

### PR DESCRIPTION
**Pull request by baytan0720**
### What problem does this PR solve?

Issue Number: #2587  

Problem Summary:

add bs update volume flatten

### What is changed and how it works?

What's Changed:

modify tools-v2/pkg/cli/command/base.go
create tools-v2/pkg/cli/command/curvebs/update/volume/volume.go
create tools-v2/pkg/cli/command/curvebs/update/volume/flatten/flatten.go

How it Works:

Usage:
```bash
curve bs update volume flatten
```

Output:
```
+------+--------------------------------------+---------+            
| USER |               TASK ID                | RESULT  |            
+------+--------------------------------------+---------+            
| root | d26e27a8-fcbd-4f7a-adf8-53795217cbb0 | success |            
+------+--------------------------------------+---------+  
```

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by baytan0720** [comment]
cicheck

File: tools-v2/pkg/cli/command/curvebs/update/volume/flatten/flatten.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		return err
+	}
+	row := make(map[string]string)
+	row[cobrautil.ROW_USER] = fCmd.user
+	row[cobrautil.ROW_TASK_ID] = fCmd.taskID
+
+	if payload["Code"] != "0" {
+		row[cobrautil.ROW_RESULT] = "failed"
+	} else {
+		row[cobrautil.ROW_RESULT] = "success"
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/update/volume/flatten/flatten.go:111]:
```suggestion
		row[cobrautil.ROW_RESULT] = cobrautil.ROW_VALUE_FAILED
	} else {
		row[cobrautil.ROW_RESULT] = cobrautil.ROW_VALUE_SUCCESS
```

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

File: tools-v2/pkg/cli/command/curvebs/update/volume/flatten/flatten.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	}
+	payload := map[string]any{}
+	if err := json.Unmarshal([]byte(result), &payload); err != nil {
+		return err
+	}
+	row := make(map[string]string)
+	row[cobrautil.ROW_USER] = fCmd.user
+	row[cobrautil.ROW_TASK_ID] = fCmd.taskID
+
+	if payload["Code"] != "0" {
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/update/volume/flatten/flatten.go:107]: defined as constant

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Comment by baytan0720** [comment]
cicheck

**Review by Cyber-SiKu — APPROVED** [review]

**Review by caoxianfei1 — APPROVED** [review]

**Review by caoxianfei1 — APPROVED** [review]

**Pull request merged**